src/read.cpp

```diff
@@ -119,7 +119,9 @@ RObject read_tokens(List sourceSpec, List tokenizerSpec, ListOf<List> colSpecs,
     collectors[t.col()]->setValue(t.row(), t);
     i = t.row();
   }
-  progressBar.show(tokenizer->progress());
+  if(progress){
```

Comment:
Could you please use house style? i.e. space after `if` and no braces for simple expression.